suite-native/module-device-settings/src/components/DevicePinProtectionCard.tsx

```diff
@@ -1,45 +1,78 @@
 import { useSelector } from 'react-redux';
 
-import { selectIsDeviceProtectedByPin, selectSelectedDevice } from '@suite-common/wallet-core';
-import { Box, CardWithIconLayout, HStack, Text, VStack } from '@suite-native/atoms';
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    selectHasRunningDiscovery,
+    selectIsDeviceProtectedByPin,
+    selectSelectedDevice,
+} from '@suite-common/wallet-core';
+import { InlineAlertBoxProps } from '@suite-native/atoms';
 import { Translation } from '@suite-native/intl';
+import {
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+
+import { SettingsItemCard } from './SettingsItemCard';
 
-import { DevicePinActionButton } from './DevicePinActionButton';
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes.DevicePinProtectionStack
+>;
```

Comment:
This component is mounted in `DeviceSettings` route, so the props should be:

```suggestion
type NavigationProp = StackNavigationProps<
    DeviceSettingsStackParamList,
    DeviceSettingsStackRoutes.DeviceSettings
>;
```

Reply:
🙈 
https://github.com/trezor/trezor-suite/pull/19638/commits/0250117056ee88489b3a507e1ed8b2a6e7ee5a75